messages/pt.json

```
@@ -1,4 +1,7 @@
 {
+  "admin/editor.form.title": "Formulário",
+  "admin/editor.form.entity.title": "Entidade",
+  "admin/editor.form.schema.title": "Schema",
```

Shouldn't it be Esquema instead of Schema?
```suggestion
  "admin/editor.form.schema.title": "Esquema",
```

---

I verified the PT docs of MasterData and it uses the word "esquema" twice and "schema" 29 times.

---

i will approve, I think it would be nice to talk to the Vladimir team. They have helped the financial people with the translations of terms...